src/_pytest/python.py

```diff
@@ -1535,7 +1535,7 @@ def setup(self) -> None:
         if isinstance(self.parent, Instance):
             self.parent.newinstance()
             self.obj = self._getobj()
-        fixtures.fillfixtures(self)
+        self._request._fillfixtures()
```

Comment:
This change shouldn't matter I think because the class here is `Function` which always has `_request` defined, so `fixtures.fillfixtures` does nothing for it except calling `self._request._fillfixtures()`.